src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -214,6 +213,41 @@ public virtual async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAc
                 var tasks = tasksResponse?.Tasks.Where(t => t.ItemsDueCount > 0 && t.Type != "AgreementToSign").ToList() ?? new List<AccountTask>();
                 var showWizard = userResponse.User.ShowWizard && userRoleResponse.UserRole == Role.Owner;
 
+
+                var apprenticeshipResponse = await _mediator.SendAsync(new Queries.GetApprenticeship.GetApprenticeshipRequest
+                {
+                    AccountId = accountResponse.Account.Id
+                });
+
+                int cohortsCount =0, draftApprenticeshipCount =0;
+                CohortSummary singleCohort = new CohortSummary();
+                DraftApprenticeshipDto singleDraftApprenticeship = new DraftApprenticeshipDto();

```

Comment:
We shouldn't be using the model from another domain here.  This dto object should be mapped to a model in our domain (SFA.DAS.EmployerAccounts.Models) at the point the service is called.

Reply:
Draft is the state of an Apprenticeship so we shouldn't be referring to Draft apprentices here we should have a collection of Apprenticeships which had a status of draft.

Reply:
ViewModel mapped from DomainModel and  the mapping is AccountCohort -> Apprenticeships -> TrainingProvider